requestor-tutorials/service-development/service-example-2-managed-erigon.md

```diff
@@ -1,4 +1,64 @@
+---
+description: Ethereum client as a service, implemented as a self-contained runtime
+---
+
 # Service Example 2: Managed Erigon
 
-NOTE this shall **not** be included in handbook by 16th June.
+The purpose of this example is to demonstrate building a Golem service using a dedicated, self-contained runtime. It features an Ethereum network client, Erigon (f.k.a. Turbo-geth) started on provider nodes on user's demand. Interaction with the service is *not* facilitated by Golem though.
+
+{% hint style="info" %}
+This example illustrates following Golem features & aspects:
+
+* Dedicated service runtime implemented with [ya-runtime-sdk](https://github.com/golemfactory/ya-runtime-sdk)
+* Service management using [yapapi-service-manager](https://github.com/golemfactory/yapapi-service-manager)
+{% endhint %}
+
+{% hint style="info" %}
+All code for this example can be found in this repository: [https://github.com/golemfactory/yagna-service-erigon/](https://github.com/golemfactory/yagna-service-erigon/)
+{% endhint %}
+
+## Architecture overview
+
+![](erigon_architecture.jpg)
+
+The application consist of three major components:
+
+1. **User interface**  
+   A browser-based interface written in TypeScript using [React](https://reactjs.org/). It allows the end user to interact with the application, i.e. start an Erigon node after authenticating with their [Metamask](https://metamask.io/) wallet, and later manage their node(s).
+2. **Requestor server**  
+   A Python application developed using [Quart](https://pgjones.gitlab.io/quart/) and [yapapi-service-manager](https://github.com/golemfactory/yapapi-service-manager). It acts both as an HTTP server handling requests from the user interface, and as a requestor agent submitting tasks to the Golem network.
```

Comment:
```suggestion
   A Python application developed using [Quart](https://pgjones.gitlab.io/quart/) and [yapapi-service-manager](https://github.com/golemfactory/yapapi-service-manager). It acts both as an HTTP server - handling requests from the user interface, and as a requestor agent - submitting tasks to the Golem network.
```

Reply:
I'm not sure if that's a correct use of dashes (actually, these are hyphens but never mind that). :thinking:

Reply:
```suggestion
   A Python application developed using [Quart](https://pgjones.gitlab.io/quart/) and [yapapi-service-manager](https://github.com/golemfactory/yapapi-service-manager). It acts both as an HTTP server (handling requests from the user interface), and as a requestor agent (submitting tasks to the Golem network).
```